pkg/app/app.go

```diff
@@ -201,7 +201,25 @@ func CreateApp() App {
 
 	log.Debug("Connecting Registry")
 	for _, config := range app.config.GetSubConfigArray("registry") {
-		reg, err := registries.NewRegistry(config, app.config.GetString("openshift.namespace"))
+		c := registries.Config{
+			URL:        config.GetString("url"),
+			User:       config.GetString("user"),
+			Pass:       config.GetString("pass"),
+			Org:        config.GetString("org"),
+			Tag:        config.GetString("tag"),
+			Type:       config.GetString("type"),
+			Name:       config.GetString("name"),
+			Images:     config.GetSliceOfStrings("images"),
+			Namespaces: config.GetSliceOfStrings("namespaces"),
+			Fail:       config.GetBool("fail_on_error"),
+			WhiteList:  config.GetSliceOfStrings("white_list"),
+			BlackList:  config.GetSliceOfStrings("black_list"),
+			AuthType:   config.GetString("auth_type"),
+			AuthName:   config.GetString("auth_name"),
+			Runner:     config.GetString("runner"),
```

Comment:
@djzager Making sure this is what you were expecting.

Reply:
Why are we copying configs everywhere now? Feels like I missed something.